elements/lisk-chain/test/unit/block_header.spec.ts

```diff
@@ -104,9 +114,9 @@ describe('block_header', () => {
 				expect(blockHeader.generatorAddress).toEqual(data.generatorAddress);
 				expect(blockHeader.previousBlockID).toEqual(data.previousBlockID);
 				expect(blockHeader.stateRoot).toEqual(data.stateRoot);
+				expect(blockHeader.validatorsHash).toEqual(data.validatorsHash);
+				expect(blockHeader.aggregateCommit).toEqual(data.aggregateCommit);
```

Comment:
maybe expect to match other header properties as well
```suggestion
				expect(blockHeader.validatorsHash).toEqual(data.validatorsHash);
				expect(blockHeader.aggregateCommit).toEqual(data.aggregateCommit);
				expect(blockHeader.maxHeightPrevoted).toEqual(data.maxHeightPrevoted);
				expect(blockHeader.maxHeightGenerated).toEqual(data.maxHeightGenerated);
				expect(blockHeader.assetsRoot).toEqual(data.assetsRoot);
```